app/bundles/ApiBundle/Config/services.php

```diff
@@ -21,5 +21,8 @@
     $services->load('Mautic\\ApiBundle\\', '../')
         ->exclude('../{'.implode(',', array_merge(MauticCoreExtension::DEFAULT_EXCLUDES, $excludes)).'}');
 
+    $services->load('Mautic\\ApiBundle\\Controller\\', '../Controller')
+        ->tag('controller.service_arguments');
+
```

Comment:
This could be replaced by removing this line: https://github.com/mautic/mautic/blob/5.x/app/bundles/CoreBundle/DependencyInjection/MauticCoreExtension.php#L28

Reply:
Sure, will remove that line later. If i remove right now the whole container could not be built, because controllers are not in the container.

Reply:
@biozshock is this conversation still relevant?